fmriprep/workflows/epi.py

```diff
@@ -138,6 +138,17 @@ def ref_epi_t1_registration(reportlet_suffix, inv_ds_suffix, name='ref_epi_t1_re
 
     explicit_mask_epi = pe.Node(fsl.ApplyMask(), name="explicit_mask_epi")
 
+    if settings['freesurfer']:
+        bbregister = pe.Node(
+            BBRegisterRPT(
+                contrast_type='t2',
+                init='header',
+                registered_file=True,
+                out_fsl_file=True,
```

Comment:
Running BBRegisterRPT, I'm running into the issue that `registered_file` and `out_reg_file` are undefined in the outputs, but not in the command (which [uses `_list_outputs`](https://github.com/nipy/nipype/blob/86cd8ffb6dae1362a07ed8e9c79d583db87780eb/nipype/interfaces/freesurfer/preprocess.py#L1043-L1051) to build the command). Neither exists in the working directory. Weirdly, `out_fsl_file` is defined and exists, so it doesn't seem like it should be an mixin/MRO issue.

```RestructuredText
Node: sub-387 (ref_epi_t1_registration (bbregister (registration)
=================================================================

 Hierarchy : workflow_enumerator.sub-387.ref_epi_t1_registration.bbregister
 Exec ID : bbregister.a2

Original Inputs
---------------

* args : <undefined>
* compress_report : auto
* contrast_type : t2
* environ : {'SUBJECTS_DIR': '/opt/freesurfer/subjects'}
* epi_mask : <undefined>
* generate_report : True
* ignore_exception : False
* init : header
* init_reg_file : <undefined>
* intermediate_file : <undefined>
* out_fsl_file : True
* out_reg_file : <undefined>
* out_report : report.svg
* reg_frame : <undefined>
* reg_middle_frame : <undefined>
* registered_file : True
* source_file : /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/explicit_mask_epi/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked.nii.gz
* spm_nifti : <undefined>
* subject_id : sub-387
* subjects_dir : /out/freesurfer
* terminal_output : allatonce

Execution Inputs
----------------

* args : <undefined>
* compress_report : auto
* contrast_type : t2
* environ : {'SUBJECTS_DIR': '/out/freesurfer'}
* epi_mask : <undefined>
* generate_report : True
* ignore_exception : False
* init : header
* init_reg_file : <undefined>
* intermediate_file : <undefined>
* out_fsl_file : True
* out_reg_file : <undefined>
* out_report : report.svg
* reg_frame : <undefined>
* reg_middle_frame : <undefined>
* registered_file : True
* source_file : /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked.nii.gz
* spm_nifti : <undefined>
* subject_id : sub-387
* subjects_dir : /out/freesurfer
* terminal_output : allatonce

Execution Outputs
-----------------

* min_cost_file : <undefined>
* out_fsl_file : /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked_bbreg_sub-387.mat
* out_reg_file : <undefined>
* out_report : /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/report.svg
* registered_file : <undefined>

Runtime info
------------

* command : bbregister --t2 --init-header --fslmat /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked_bbreg_sub-387.mat --reg /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked_bbreg_sub-387.dat --o /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked_bbreg.nii.gz --mov /scratch/workflow_enumerator/sub-387/ref_epi_t1_registration/_epi_..data..sub-387..func..sub-387_task-Beast_run-3_bold.nii.gz/bbregister/sub-387_task-Beast_run-3_bold_mcf.nii.gz_mean_reg_masked.nii.gz --s sub-387
* duration : 67.60915
* hostname : 9c7c0a3e6622

Terminal output
~~~~~~~~~~~~~~~
```

@chrisfilo Anything obvious stick out to you?